src/importexport/guitarpro/internal/importgtp.cpp

```diff
@@ -226,12 +227,11 @@ String GuitarPro::readPascalString(int n)
     if (n - l > 0) {
         skip(n - l);
     }
-
-    try {
+    std::string_view sw{ &s[0] };
+    if (utf8::is_valid(sw)) {
         return String::fromUtf8(&s[0]);
-    } catch (...) {
-        return String::fromAscii(&s[0]);
     }
+    return String::fromAscii(&s[0]);
```

Comment:
I would recommend passing the length to avoid having to calculate it.
(to std::string_view  and String::from...)